data_sources/crowdstrike_processrollup2.yml

```diff
@@ -96,6 +96,7 @@ field_mappings:
   mapping:
     CommandLine: Processes.process
     ImageFileName: Processes.process_path
+    ImageFileName|endswith: Processes.process_name
```

Comment:
@P4T12ICK Do we need both of these ImageFileName ?

Reply:
Yes this is needed.